mic/utils/fs_related.py

```diff
@@ -1133,9 +1133,14 @@ def cleanup(self):
         else:
             os.unlink(self.device)
 
+DEVICE_LOCKFILE = "/var/lock/__mic_loopdev.lock"
+
 def get_loop_device(losetupcmd, lofile):
+    global DEVICE_LOCKFILE
+
     import fcntl
-    fp = open("/var/lock/__mic_loopdev.lock", 'w')
+    makedirs(os.path.dirname(DEVICE_LOCKFILE))
```

Comment:
This will fail (or should be created at the non legacy path when /var/run doesn't exist.

Reply:
I was just about to test it and yeah sadly makedirs didn't follow the symlink as I expected. Fixed.